test/run_tests.sh

```diff
@@ -184,6 +184,7 @@ function run_xla_op_tests {
   run_use_bf16 "$CDIR/test_data_type.py"
   run_xla_ir_debug "$CDIR/test_env_var_mapper.py"
   run_xla_hlo_debug "$CDIR/test_env_var_mapper.py"
+  run_xla_hlo_debug "$CDIR/test_stablehlo_dump.py"
```

Comment:
let's create a dir called `stablehlo` or `inference` and put tests under that dir, there are too many tests under `test/` dir right now

Reply:
Sure, that's a good idea, let me create a new folder, maybe just use `stablehlo`